### PR TITLE
Feature task board

### LINKED ISSUE #31: Task Board columns
Add 3 vertical columns to the screen. The columns are to contain tasks. The leftmost task should have a dummy '+' button that will have its functionality linked up later on. The 'Today's Tasks' screen has a similar list that Yuno mentioned could be heavily reused. in our screen

### LINKED ISSUE #32: Task Board Drag and Drop
Add the ability to drag tasks between the 3 vertical columns in the Trello board screen. The ability to drag/reorder tasks within one column can also be added but depending on how much time we have this can be left. This functionality should also be linked to firebease so that the changes are persisted.

### LINKED ISSUE #33: Task Board Task Sorting
Change the order that the tasks are sorted within their vertical column. Examples of the way that tasks can be sorted include due date and class priority.

### PR DESCRIPTION
**Pull request by TessaGush**
Created the column component. 
CSS styling.

### CODE REVIEW

File: src/components/global/Column.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import React from "react";
+// import { Link } from "react-router-dom";
```
Comment (saaaaraaaah) [inline src/components/global/Column.jsx]: Some commented out code of unused imports. Should be removed if they wont be used

Comment (ireb15) [reply src/components/global/Column.jsx]: Agreed

**Review by ireb15 — APPROVED** [review]
Looks good to me! Great work

**Review by saaaaraaaah — APPROVED** [review]
Looks good to me!

**Review by ireb15 — APPROVED** [review]

**Pull request merged**